src/actions/guides/database/validating-saving.cr

```diff
@@ -145,12 +145,12 @@ class Guides::Database::ValidatingSaving < GuideAction
     class Users::Create < BrowserAction
       route do
         # params will have the form params sent from the HTML form
-        SaveUser.create(params) do |form, user|
+        SaveUser.create(params) do |operation, user|
           if user # if the user was saved
             redirect to: Home::Index
           else
             # re-render the NewPage so the user can correct their mistakes
-            html NewPage, user_form: form
+            html NewPage, user_operation: operation
```

Comment:
On line 122, this `NewPage` actually `needs operation`. So this should just be `operation`. However, maybe it should say `save_user: operation`, and then line 122 changes to `needs save_user : SaveUser`?

Reply:
Thanks for catching `needs operation` part!

And I've also had similar thoughts when noticed `save_operation: operation` in other place, while all other examples just using `operation: operation`.

Maybe for some operations it would make sense to use  a name different from just `operation` to clarify things.
Maybe always doing `operation: operation` is fine, at leas in all provided examples.

I think we need to capture a list of used names and vote which option looks best in each case.

Reply:
I personally don't like `operation: operation` because it almost feels like you're locked in to that. I think changing it up from example to example helps to make it clear that this keyword is determined by you (the developer), and not the framework.

Reply:
Maybe @paulcsmith or @edwardloveall have some insight here ^

Reply:
I think almost always it's better to have a more specific name instead of `operation`. I like the pattern we've used before of `verb_noun` like `save_user`. It would be great to find examples that made sense in context to further hint to readers what an operation does.

Reply:
I agree. In my own projects I always use the operation name. The generators use `operation: operation` because it was easier but in general I use `needs verb_noun`. I'm 100% ok using `operation` in the block arg though, or even `op` since it is used so much.

I think for examples though using `operation` is better for clarity. So I like this change 👍

Reply:
Updated to suggested more clear names.